packages/components/src/components/button-group/types.ts

```diff
@@ -0,0 +1,11 @@
+import { Generic } from '@a11y-ui/core';
+
+type RequiredProps = unknown;
```

Comment:
Ist `unknown` eigentlich richtig? Warum nicht `{}`?

Bei Verkettung des Typs kommt es so zur Warnung: "'unknown' is overridden by other types in this intersection type".

Reply:
Mal `{}` statt `unknown` ausprobieren. Früher hat des Lint-Errors geworfen.

Reply:
Die Klammern erzeugen von Prettier einen Fehler:
```sh
Don't use `{}` as a type. `{}` actually means "any non-nullish value".
- If you want a type meaning "any object", you probably want `object` instead.
- If you want a type meaning "any value", you probably want `unknown` instead.
- If you want a type meaning "empty object", you probably want `Record<string, never>` instead.
- If you really want a type meaning "any non-nullish value", you probably want `NonNullable<unknown>` instead.eslint[@typescript-eslint/ban-types](https://typescript-eslint.io/rules/ban-types)
```

Ich würde dem `unknown` das `object` vorziehen, da `unknown` auch primitive Typen beinhaltet.